### ISSUE TITLE
Low confidence score with high average precision

### ISSUE THREAD

**Issue by kapil-varshney** [DeNA/PyTorch_YOLOv3#63]
@hirotomusiker 
I am training Yolov3 on a custom dataset with single class. The dataset has 1240 images of size 256x256 with 2029 class instances of size ~35x35. I have generated anchor boxes according to my dataset using k-means.
I trained it for 10+100 epochs at lr=0.001 (cyclic learning rate).

I am getting low confidence scores (0.1~0.45) for true positives, but relatively high average precision (~88 at conf_thresh=0.1).
Is this an expected behavior or there could be something I am not doing right?
